src/vue/forms/TfaForm.vue

```diff
@@ -0,0 +1,237 @@
+<template>
+  <form
+    class="app-form tfa-form"
+    @submit.prevent="isFormValid() && enableTfa()"
+  >
+    <template v-if="!form.factor">
+      <div class="app__form-row">
+        <div class="app__form-field">
+          <input-field
+            v-model="form.password"
+            @blur="touchField('form.password')"
+            id="tfa-password"
```

Comment:
why do we need an id here?

Reply:
For the field bindings in unit tests

https://github.com/tokend/web-client/blob/86707d8eea73a3b56e918e00b1d8fc2c374e59be/src/vue/forms/LoginForm.vue#L8 https://github.com/tokend/web-client/blob/86707d8eea73a3b56e918e00b1d8fc2c374e59be/src/vue/forms/LoginForm.vue#L20

https://github.com/tokend/web-client/blob/86707d8eea73a3b56e918e00b1d8fc2c374e59be/src/vue/forms/LoginForm.spec.js#L57-L60